perun-core/src/main/java/cz/metacentrum/perun/core/api/AuthzResolver.java

```diff
@@ -39,6 +40,20 @@ public static boolean authorizedExternal(PerunSession sess, String policyDefinit
 		return AuthzResolverBlImpl.authorized(sess, policyDefinition, objects);
 	}
 
+	/**
+	 * Checks if the principal is authorized.
+	 * Used when there are no PerunBeans needed for authorization.
+	 * This method should be accessed through external components.
+	 *
+	 * @param sess PerunSession which contains the principal.
+	 * @param policyDefinition of policy which contains authorization rules.
+	 * @return true if the principal has particular rights, false otherwise.
+	 * @throws PolicyNotExistsException when the given policyDefinition does not exist in the PerunPoliciesContainer.
+	 */
+	public static boolean authorizedExternal(PerunSession sess, String policyDefinition) throws PolicyNotExistsException {
```

Comment:
I think it is not necessary to provide this version of the method in RPC.

Reply:
Erased